This patch fixes the quotation for a few f-string expressions that Python-3.11 cannot parse.
Author: Stamen Miroslavov <stamen.miroslavov@bsc.es>
Barcelona Supercomputing Center 2026
diff -ruNBbw a/src/preprocessor.py b/src/preprocessor.py
--- a/src/preprocessor.py	2026-10-21 12:54:55.487448045 +0200
+++ b/src/preprocessor.py	2026-10-21 13:39:21.003944628 +0200
@@ -1635,8 +1635,8 @@
 
                 d.update({'project_id': var.translation.convention})
                 d.update({'path': var.dest_path})
-                d.update({'time_range': f'{util.cftime_to_str(ds_match.time.values[0]).replace('-',':')}-'
-                                        f'{util.cftime_to_str(ds_match.time.values[-1]).replace('-',':')}'})
+                d.update({'time_range': f"{util.cftime_to_str(ds_match.time.values[0]).replace('-',':')}-"
+                                        f"{util.cftime_to_str(ds_match.time.values[-1]).replace('-',':')}"})
                 d.update({'standard_name': ds_match[var.name].attrs['standard_name']})
                 cat_entries.append(d)
 
